pkg/controller/networkpolicy/validate_test.go

```diff
@@ -30,15 +30,21 @@ import (
 	"antrea.io/antrea/pkg/features"
 )
 
-func TestValidateAntreaPolicy(t *testing.T) {
-	allowAction := crdv1alpha1.RuleActionAllow
-	passAction := crdv1alpha1.RuleActionPass
-	int32For80 := int32(80)
+var (
+	query       = crdv1alpha1.IGMPQuery
+	report      = crdv1alpha1.IGMPReportV1
+	allowAction = crdv1alpha1.RuleActionAllow
+	passAction  = crdv1alpha1.RuleActionPass
+	int32For80  = int32(80)
```

Comment:
use some significant variable name

Reply:
These variables was  already present in the test code

Reply:
You can refine the name if it's not proper, I would suggest to name it `port80`.